web/help/GuideUnit1.md

```diff
@@ -124,14 +121,18 @@ nametag = lettering("Camille")
 You've used a new function: **`lettering`**.  This function describes a
 picture with letters (or any other kind of text) on it.
 
+Did you notice the word **Compiling** popped up and went away? This is the computer reading your instructions to make sure they are correct before executing them.
+
 Next, you can add a border to your nametag.  You might be tempted to add
 a new line like `nametag = ...` to your code, but you can't! Remember,
 your code is like a dictionary, and each definition in it should give
 the whole definition for that word.  To include a second shape in your
-nametag, you'll use **`&`**, which you can read as "and" or "in front
+nametag, you'll use an *operator*, **`&`**, which you can read as "and" or "in front
```

Comment:
It might be a good idea to introduce operators here.  If so, though, I think a lot more needs to be said.  Students should be reminded of the operators they already know about on numbers, like addition.

This is already discussed in a collapsible box under the "Expressions and types" section later on.  I think it's a bit of a balance.  Right here, we're in a hurry to get students to writing their own creative programs as soon as possible, so whatever absolutely needs to be said should go here, but any long-winded asides can wait until later.

Reply:
Noted